doc/whats_new/v0.21.rst

```diff
@@ -523,3 +523,14 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :issue:`12328` by :user:`Nicolas Hug <NicolasHug>`
+- Created two new multioutput estimator tests:
+  :func:`~utils.estimator_checks.check_classifier_multioutput` and
+  :func:`~utils.estimator_checks.check_regressor_multioutput`.
+  Added them to :func:`~utils.estimator_checks._yield_classifier_checks` and
+  :func:`~utils.estimator_checks._yield_regressor_checks` respectively.
+  Added three intermediate private classes (
+  :class:`~linear_model.coordinate_descent._BaseLinearModelCV`,
+  :class:`~linear_model.least_angle._BaseLars`,
```

Comment:
This whatsnew entry is rather verbose.
I would remove the mention to `_yield_classifier_checks` and the mention to private classes. These are implementation details that should not affect users.

Reply:
Changed! :)